common/persistence/sql/sqlplugin/mysql/session/session_test.go

```diff
@@ -195,5 +195,8 @@ func buildExpectedURLParams(attrs map[string]string, isolationKey string, isolat
 		result[k] = []string{v}
 	}
 	result["rejectReadOnly"] = []string{"true"}
+	if dbKind != sqlplugin.DbKindVisibility {
```

Comment:
Since this is the second place we make that check, could we extract a function like `allowsParamInterpolation`?

Reply:
That seems overkill for a single if statement that will only be changed when we fix the MySQL 8 visibility implementation

Reply:
I think it's definitely worth it since it's a decision that's being made in two places and there's a risk of only changing one (since the next person might not know about the other place).

Reply:
I'm ok either way, but I noticed something else: is `isolationKey` and `isolationValue` used anywhere in this function? Can we remove them?

Reply:
I ended up making the change after all.

As for `isolation{Key,Value}` they're used on line 193 just above the fold